Firestore/core/src/firebase/firestore/core/sync_engine.cc

```diff
@@ -281,13 +281,19 @@ void SyncEngine::HandleRejectedListen(TargetId target_id, Status error) {
     // store's invariants with another method.
     NoDocument doc(limbo_key, SnapshotVersion::None(),
                    /* has_committed_mutations= */ false);
-    DocumentKeySet limbo_documents = DocumentKeySet{limbo_key};
-    RemoteEvent event{SnapshotVersion::None(), /*target_changes=*/{},
-                      /*target_mismatches=*/{},
-                      /*document_updates=*/{{limbo_key, doc}},
+
+    // Explicitly instantiate these to work around a spurious "chosen
+    // constructor is explicit in copy-initialization" error in the RemoteEvent
+    // constructor on GCC 4.9.
```

Comment:
Nit: is the error in GCC 4.9? The log Jon posted seems to indicate GCC 4.8:
```
/usr/bin/../lib/gcc/x86_64-linux-gnu/4.8/../../../../include/c++/4.8/bits/unordered_map.h:138:7: note: explicit constructor declared here
      unordered_map(size_type __n = 10,
```

Reply:
Updated.